CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 - properties are now sorted, and iterating over properties will always yield
   them sorted by the associated key.
+- Added read and write support for Amber Restart (.ncrst) files.
```

Comment:
this should be in the section below "changes in supported formats"

Reply:
Oh, I missed that section.